tests/Integration/Behaviour/Features/Scenario/Product/update_prices_without_states.feature

```diff
@@ -0,0 +1,278 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s product --tags update-prices-without-states
+@restore-products-before-feature
+@reboot-kernel-before-feature
+@update-prices-without-states
+@reboot-kernel-after-feature
+@restore-all-tables-after-feature
+Feature: Update product price fields from Back Office (BO) when default country has no states.
```

Comment:
Testing a country with states makes sense, however we don't really test the use case that interests us here When a country has multiple states then the default one is the first one To test this it's worth having a new feature where you create/edit a country with states and remove the first one to ensure the default tax rate is then updated